tests/std/tests/P0088R3_variant/test.cpp

```diff
@@ -7067,6 +7067,15 @@ namespace msvc {
             static_assert(!std::is_copy_constructible_v<std::variant<volatile S>>);
         }
     } // namespace vso508126
+
+    namespace DevCom1031281 {

```

Comment:
This is missing a short description of the bug. Yes, it's in the bug database, but bug databases don't last as long as code.